services/network/network.go

```diff
@@ -796,7 +800,7 @@ type SecurityRulePropertiesFormat struct {
 	// Access - The network traffic is allowed or denied. Possible values include: 'SecurityRuleAccessAllow', 'SecurityRuleAccessDeny'
 	Access SecurityRuleAccess `json:"access,omitempty"`
 	// Priority - The priority of the rule. The value can be between 100 and 4096. The priority number must be unique for each rule in the collection. The lower the priority number, the higher the priority of the rule.
```

Comment:
>100 and 4096 [](http://example.com/codeflow?start=66&length=12)

0 to 65536

Reply:
In Azure it's 100 to 4096. I don't think we're failing operations if it's out of this range before network controller though